packages/postgres-database/src/simcore_postgres_database/migration/versions/99f9b200801c_adde_sharing_networks_table.py

```diff
@@ -0,0 +1,33 @@
+"""adde sharing networks table
+
+Revision ID: 99f9b200801c
+Revises: a6b281f3ee15
+Create Date: 2022-03-07 08:54:31.894295+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '99f9b200801c'
+down_revision = 'a6b281f3ee15'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('sharing_networks',
+    sa.Column('project_uuid', sa.String(), nullable=False),
+    sa.Column('networks_with_aliases', postgresql.JSONB(astext_type=sa.Text()), server_default=sa.text("'{}'::jsonb"), nullable=False),
+    sa.ForeignKeyConstraint(['project_uuid'], ['projects.uuid'], name='fk_sharing_networks_project_uuid', onupdate='CASCADE', ondelete='CASCADE'),
```

Comment:
can you check the naming practices for foreign key please?
it should be ```fk_THISTABLENAME_FIELD_OTHERTABLENAME

Reply:
Ahh good to know. Moving to `fk_project_networks_project_networks_projects`.